views/pages/ViewChartPage.jsx

```diff
@@ -66,6 +86,7 @@ const ViewChartPage = () => {
           onSubmit={handleSubmit(handleFormSubmit)}
           classes={classes}
           control={control}
+          siteOptions={siteOptions}
```

Comment:
should we pass the sites array to this form and let the form know how to create the form UI from them?

Reply:
We can pass the array but then we would be doing the transformation on the form page, our current pattern is to that the Parent page handles all of that and then passes it to the child. 

With react select we need to create an array of objects with the keys label and value, if we pass the array to the form page then we would redo the transformation array.

Is this at what you're hinting towards..?

Reply:
Yeah, there's also the consideration of the default form values. So i've opted for another path. [218cfc9](https://github.com/AMP-SCZ/dpdash/pull/441/commits/218cfc99b61b8fbafa89ab15275c64beca1f0b10)